app/controllers/services_controller.rb

```diff
@@ -59,11 +59,11 @@ def hidden_user_explanation
            formats: [:text],
            layout: false,
            locals: {
-             name_to: info_request.user_name,
+             name_to: info_request.user_name.html_safe,
```

Comment:
I think we should qualify (in the commit message) that as this is rendering `text/plain`, malicious user input won't be rendered as HTML so `<script>` tags can't be injected.